appveyor.yml

```diff
@@ -4,6 +4,7 @@ image: Visual Studio 2015
 
 cache:
   - '%LocalAppData%\Microsoft\dotnet'
+  - '%HOMEDRIVE%%HOMEPATH%\.nuget\packages'
```

Comment:
New packages have newer versions. Our dependencies are pretty conservative and old version should not be replaced in place, although it's possible, i.e. if dotnet team decided to remove some old package and publish a newer version reusing the same version. But I don't think it's a real scenario.